api/camunda/camunda-openapi.yaml

```diff
@@ -1501,25 +1745,45 @@ components:
           format: int64
         tenantId:
           type: string
-    AuthorizationAssignRequest:
+    AuthorizationPatchRequest:
       type: object
       properties:
-        ownerKey:
-          type: string
-        ownerType:
+        action:
+          description: Indicates if permissions should be added or removed.
           type: string
           enum:
+            - ADD
+            - REMOVE
+        resourceType:
+          description: The type of resource to add/remove perissions to/from.
+          enum:
+            - AUTHORIZATION
+            - MESSAGE
+            - JOB
+            - APPLICATION
+            - TENANT
+            - DEPLOYMENT
+            - PROCESS_DEFINITION
+            - USER_TASK
+            - DECISION_REQUIREMENTS_DEFINITION
+            - DECISION_DEFINITION
+            - USER_GROUP
             - USER
             - ROLE
-            - GROUP
-        resourceKey:
-          type: string
-        resourceType:
-          type: string
         permissions:
           type: array
           items:
-            type: string
+            properties:
+              permissionType:
+                enum:
+                  - CREATE
+                  - READ
+                  - UPDATE
+                  - DELETE
+              resourceIds:
+                type: array
+                items:
+                  type: string
```

Comment:
category: missing descriptions

owner: me, initially

not sure what these look like in the UI, and the descriptions might not be needed?

Reply:
They're relatively self-explanatory, but for consistency's sake we should add descriptions. I will track down the originator and tag them to follow up.

Reply:
@remcowesterhoud -- in [a recent PR](https://github.com/camunda/camunda/pull/21910) to the C8 REST API, you added or updated the following, each of which is missing `description` properties in the OpenAPI spec:

- [AuthorizationPatchRequest](https://github.com/camunda/camunda/blob/55512df08633ee6b01a3b84af1999919f6e467c4/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L1791-L1829) schema

The descriptions are important because the docs ingest the spec from the `camunda` repo, and generate our API docs from it. I'm tagging you here in the `camunda-docs` repo, instead of the `camunda` repo, because that's where we (DevEx) noticed the descriptions missing.

Please add descriptions to these items, in the [C8 REST OpenAPI spec](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml). Your changes will eventually propagate downstream to the docs here.

For guidance on writing the descriptions, [see here](https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension). If you have any questions, feel free to ask. 

Thank you!